File: classwork/Secondlargest.java
import java.util.Scanner;
public class Secondlargest{

public static void main(String [] args){
	Scanner input = new Scanner(System.in);
	int [] value = new int[5];
	 	int largest = value[0];
		int secondLargest = -999999999;
	
	for(var i = 0; i < value.length; i++){
		System.out.println("Enter the set of numbers");
		value[i] = input.nextInt();
		if(value[i] > largest){secondLargest = largest; largest = value[i];}
		if(largest  > value[i] && value[i] > secondLargest) secondLargest =  value[i];
	}
	System.out.println(largest);
	System.out.println(secondLargest + " secondLargest");
}

}

File: classwork/secondLargest.js
const score = []
const prompt = require('prompt-sync')()
let largest = 0
 let secondlargest = 0
for(var count = 0; count < 8;count++){
	score[count] = prompt("Enter the scores") 
	if(score[count] > largest) largest = score[count] 

}
console.log(largest)

for(var counter = 0; counter < score.length; counter++){
	if(score[counter] == largest) continue
	if(score[counter] > secondlargest) secondlargest = score[counter]
	
}
console.log(secondlargest)
